docs/frontend.md

```diff
@@ -169,13 +165,52 @@ Custom elements provide several advantages in that they...
 - have no dependencies, limiting overall page size in the critical path
 - are portable and avoid vendor lock-in
 
-### Localization
+### Conventions
 
-See [`@18f/identity-i18n` package documentation](https://github.com/18F/identity-idp/blob/main/app/javascript/packages/i18n/README.md).
+#### Naming
+
+Each component should have a name that is used consistently in its implementation and which
+describes its purpose. This should be reflected in file names and the code itself.
+
+- ViewComponent classes should be named `[ExampleName]Component`
+- ViewComponent files should be named `app/components/[example_name]_component.rb`
+- Stylesheet files should be named `app/stylesheets/components/_[example-name].scss`
```

Comment:
This path is unclear. Should it be `app/assets/stylesheets/components/_[example-name].scss`?

Reply:
Yes, it should be that. Good catch! Updated in 89fe83f28.